en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,354 @@
+           "Namespace": "default",
+           "Replicas": 1
+            }
+          }
+        ]
+      }
+    ]
+    ```
+ 
+     <html><div class="admonition note">
+        <p class="admonition-title">Note</p>
+        <p> Super tenant users can customize the container management implementations if required. Then customize classes should be added to the `<API-M_HOME>/repository/conf/deployment.toml` file as follows. </p>   
+        
+    ``` json tab="Format"
+    [ContainerMgtConfig]
+    <Environment-type> = "<class-name>"
+    ```  
+   
+    ``` json tab="Example"
+    [ContainerMgtConfig]
+    Kubernetes = "org.wso2.carbon.apimgt.impl.containermgt.K8sManager_CustomClass"
+    ```
+  
+      </div>
+      </html>
+         
+4. Start the server and create an API in publisher
+    
+    After setting up configurations go to the <API-M-HOME>/bin directory and start the server. Then Sign in to the WSO2 API Publisher and [create a new API]({{base_path}}/learn/design-api/create-api/create-a-rest-api.md).
+    
+5. Select the clusters from environment tab
+
+    Navigate to the environment tab by clicking on the `Environments` in the navigation bar, and select the clusters to deploy the API.
+    
+     ![Environments]({{base_path}}/assets/img/learn/privatejet-mode/environment.png)
+    
+6. Go to lifecycle tab by clicking on the `Lifecycle` in the navigation bar, and publish the API.
+
+    Navigate to the lifecycle tab and publish the API. This will expose microservices as managed APIs in selected clusters.
+    
+     ![API LifeCycle]({{base_path}}/assets/img/learn/privatejet-mode/lifecycle_publish.png)
+     
+7. Go to the Developer Portal ( `https://<hostname>:9443/carbon` ) or click **View in Dev Portal** in the API Publisher and navigate to the Overview tab by clicking on the `Overview` in the navigation bar, and get the accessURL to access the deployed API Log in to the Management Console 
+   
+    To obtain the accessURL configured, Sign in to the WSO2 Developer Portal overview page. The accessURL can be seen under Gateway Environments.
+   
+    ![Developer portal - Overview]({{base_path}}/assets/img/learn/privatejet-mode/devportal.png)
+   
+8. Invoking the API
+    
+    i. Sign in to the Developer Portal ( `https://<hostname>:9443/carbon` ). Subscribe to the API and obtain an access token required to invoke the API.
```

Comment:
This url is wrong. It should be https://<hostname>:9443/devportal. Check the other place as well.

Reply:
Done